packages/react-native-version-check/src/getStoreUrl.js

```diff
@@ -19,20 +19,21 @@ export type GetStoreUrlOption = GetAppStoreUrlOption & GetPlayStoreUrlOption;
 export const getAppStoreUrl = async (
   option: GetAppStoreUrlOption
 ): Promise<string> => {
+  const opt = option;
```

Comment:
Looking at the code for `getPlayStoreUrl`, I assume this might need to be:

```suggestion
  const opt = option || {};
```